coalib/coala_modes.py

```diff
@@ -61,8 +59,10 @@ def mode_json(args, debug=False):
 
     if args.show_bears:
         try:
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, LogPrinter())
+            from coalib.parsing.FilterHelper import FilterHelper
+
+            local_bears, global_bears = FilterHelper.get_filtered_bears(
+                'language', args.filter_by_language)
```

Comment:
At the moment, there's no other way of providing languages apart from `filter_by_language`. So, to use `filter_by` over here, we need to repeat the same logic as present in coala.py